sql/moz-fx-data-shared-prod/search/mobile_search_clients_engines_sources_daily/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+---
+friendly_name: Mobile Search Clients Engines Sources Daily
+description: |
+  A daily aggregate of baseline and metrics pings that have
+  searches from each (client_id, engine, source) combination,
```

Comment:
`type` (or `search_type`) is also used as a key in the `GROUP BY`, but if I'm reading this logic correctly, each `engine` will only be associated with a `type`. If that's not correct, then this name may need to include `_type` as well.

Reply:
Search type can be determined by source and the field is a bit of an implementation detail so I don't think it's necessary to include `_type`

Reply:
LGTM. 

Just note that for `mobile` there is also `app_name`, which seems can be determined by (`source`, `client_id`, 
`engine`), I feel like `app_name` is more common dimension than `source`; however, your proposed name might be better in the spirit of consistent naming.